packages/ui/src/forum/hooks/useForumLatestPosts.ts

```diff
@@ -1,48 +1,29 @@
-import * as Apollo from '@apollo/client'
-import { useEffect, useState } from 'react'
+import { useEffect } from 'react'
 
 import { ForumPostOrderByInput } from '@/common/api/queries'
+import { A_DAY } from '@/common/constants'
 import { useLocalStorage } from '@/common/hooks/useLocalStorage'
-import { GetForumPostsDocument } from '@/forum/queries'
 
-const orderBy = [ForumPostOrderByInput.UpdatedAtDesc]
-
-const useCacheWithExpiration = (key: string): Apollo.WatchQueryFetchPolicy => {
-  const [lastCacheTime, setLastCacheTime] = useLocalStorage<string>(key)
-  const [fetchPolicy, setFetchPolicy] = useState<Apollo.WatchQueryFetchPolicy>('cache-first')
-
-  useEffect(() => {
-    const currentDate = new Date()
-
-    try {
-      if (lastCacheTime != undefined) {
-        const lastCacheDate = new Date(lastCacheTime)
-        const timeDiff = Math.abs(currentDate.getTime() - lastCacheDate.getTime())
+import { useGetForumPostsQuery } from '../queries'
+import { ForumPost, asForumPost } from '../types'
 
-        if (timeDiff < 1000 * 3600 * 24) {
-          setFetchPolicy('cache-first')
-          return
-        }
-      }
-    } catch (err) {
-      // ignore issue
-    }
-
-    setFetchPolicy('cache-and-network')
-    setLastCacheTime(currentDate.toDateString())
-  }, [lastCacheTime])
-
-  return fetchPolicy
+export type ForumPostsCache = {
+  expires: number
+  posts: ForumPost[]
 }
+const orderBy = [ForumPostOrderByInput.UpdatedAtDesc]
```

Comment:
Since `orderBy` is already here let's put the whole `variables` here too:
```suggestion
const variables = { where: { isVisible_eq: true }, orderBy: [ForumPostOrderByInput.UpdatedAtDesc], limit: 500 }
```